core/tauri-runtime-wry/src/lib.rs

```diff
@@ -3427,6 +3449,9 @@ fn create_window<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
 
   if let Some(webview) = webview {
     webviews.push(create_webview(
+      #[cfg(feature = "unstable")]
+      WebviewKind::WindowChild,
```

Comment:
Why do we want to use `WindowChild` for `WebviewWindow` ? I think we still has issues with that on macOS

Reply:
it's explained in the PR description, note this is all behind the unstable feature flag

Reply:
yeah but in the future we plan to remove this `unstable` feature, no?

Reply:
yeah when we sort all the bugs and stabilize the API (that's why i mentioned the flag)

Reply:
cool